src/components/tutor-input.cjsx

```diff
@@ -141,7 +141,7 @@ TutorDateInput = React.createClass
           weekStart={0}
         />
     else if @props.disabled and value
-      displayValue = value.toString("YYYY/MM/DD")
+      displayValue = value.toString("MM/DD/YYYY")
 
```

Comment:
I had a ticket in pivotal that said to change it to this format.  I personally prefer YYYY/MM/DD as well.  I will pull this out and put it as a constant somewhere though...